package.json

```diff
@@ -33,6 +34,7 @@
     "prop-types": "^15.6.0",
     "qs": "^6.7.0",
     "react": "~16.8.6",
+    "react-bootstrap": "^1.0.0-beta.16",
```

Comment:
Do we know how much we're increasing the size of the bundle by bringing in these libraries (and their dependencies)?  I know it's already huge thanks to plotly, but I just want to make sure we've thought about it.

Reply:
per feedback we've removed bootstrap and moved to material UI